### PR TITLE
Hybridization cleanup

### PR DESCRIPTION
**Pull request by ryantmer**
This component had already been mostly hybridized, but all of its deps are now hybrid (I think), so things can be cleaned up a bit. Also cleaned up some Travis stuff.

TODO:

- [x] Merge and release https://github.com/BrightspaceUI/tile/pull/51 and update the version here

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,5 @@
...
-      - oracle-java8-installer
-      - oracle-java8-set-default
-before_script:
-- npm run test:lint:js
-- npm run test:lint:wc
+node_js: node
 script:
-- xvfb-run wct
+- npm run lint
+- polymer test --skip-plugin sauce
```
Comment (svanherk) [inline .travis.yml]: Do we want to setup to actual run with Sauce?

Comment (ryantmer) [reply .travis.yml]: Oh yeah, might as well! Good call

**Comment by ryantmer** [comment]
@svanherk This look okay now? Note that I removed IE11 from Sauce... it simply was not cooperating 😠 But, this does continue to work in IE11 despite that... yeah idk.

**Comment by svanherk** [comment]
@ryantmer Looks good to me!  Make sure to let Josh know how much time was spent fiddling with IE11

**Pull request merged**